Core/src/org/sleuthkit/autopsy/timeline/ui/filtering/datamodel/RootFilterState.java

```diff
@@ -260,17 +249,32 @@ public boolean isSelected() {
     }
 
     @Override
-    public ReadOnlyBooleanProperty selectedProperty() {
+    public BooleanProperty selectedProperty() {
         return ALWAYS_TRUE;
     }
 
     @Override
     public void setDisabled(Boolean act) {
-        throw new UnsupportedOperationException("Not supported yet."); //To change body of generated methods, choose Tools | Templates.
+//        throw new UnsupportedOperationException("Not supported yet."); //To change body of generated methods, choose Tools | Templates.
```

Comment:
For both setDisabled and setSelected, I would removed the commented out line and leave a comment about why you are overriding the functions, but they do nothing.